@WAI/compile/mission_winorfail.sqf

```diff
@@ -0,0 +1,313 @@
+		};
+
+	} count allUnits;
+	
+	if (wai_radio_announce) then {
+		RemoteMessage = ["radio","[RADIO] " + _msgstart];
+		publicVariable "RemoteMessage";
+	} else {
+		[nil,nil,rTitleText,_msgstart,"PLAIN",10] call RE;
+	};
+	
+	clearWeaponCargoGlobal _crate;
+	clearMagazineCargoGlobal _crate;
+
+	_crate setVariable ["ObjectID","1",true];
+	_crate setVariable ["permaLoot",true];
+
+	_crate addEventHandler ["HandleDamage", {}];
+	
+	markerready = true;
+
+	while {!_start && !_timeout} do {
+
+		sleep 1;
+		_currenttime = time;
+
+		{
+			if((isPlayer _x) && (_x distance _position <= 1500)) then {
+				_start = true
+			};
+
+		} count playableUnits;
+
+		if (_currenttime - _starttime >= _timeout_time) then {
+			_timeout = true;
+		};
+
+	};
+
+	{	
+		_x enableAI "TARGET";
+		_x enableAI "AUTOTARGET";
+		_x enableAI "MOVE";
+		_x enableAI "ANIM";
+		_x enableAI "FSM";
+	} count _mission_units;
+
+	while {!_timeout && !_complete} do {
+
+		sleep 1;
+		_currenttime = time;
+		{
+			if((isPlayer _x) && (_x distance _position <= wai_timeout_distance)) then {
+				_player_near = true;
+			};
+			
+		} count playableUnits;
+
+		if (_currenttime - _starttime >= _timeout_time && !_player_near) then {
+			_timeout = true;
+		};
+		
+		call {
+
+			if (_type == "crate") exitWith {
+
+				if(wai_kill_percent == 0) then {
+
+					{
+						if((isPlayer _x) && (_x distance _position <= 20)) then {
+							_complete = true;
+						};
+					} count playableUnits;
+
+				} else {
+
+					if(((wai_mission_data select _mission) select 0) <= _killpercent) then {
+						{
+							if((isPlayer _x) && (_x distance _position <= 20)) then {
+								_complete = true;
+							};
+						} count playableUnits;
+					};
+
+				};
+
+			};
+
+			if (_type == "kill") exitWith {
+				if(((wai_mission_data select _mission) select 0) == 0) then {
+					_complete = true;
+				};
+			};
+
+			if (_type == "patrol") exitWith {
+				if(((wai_mission_data select _mission) select 0) == 0) then {
+					_complete = true;
+				};
+			};
+			
+			if (_type == "assassinate") exitWith {
+				_objectivetarget = (_this select 1) select 1;
+				{
+					_complete = true;
+					if (alive _x) exitWith {_complete = false;};
+				} count units _objectivetarget;
+			};
+
+			if (_type == "resource") exitWith {
+				_node 		= (_this select 1) select 1;
+				_resource 	= _node getVariable ["Resource", 0];
+				if (_resource == 0) then {
+					{
+						if((isPlayer _x) && (_x distance _position <= 80)) then {
+							_complete = true;
+						} else {
+							_timeout = true;
+						};
+					} count playableUnits;
+				};
+			};
+		};
+	};
+
+	if (_complete) then {
+
+		if (typeOf(_crate) in (crates_large + crates_medium + crates_small)) then {
+
+			if(wai_crates_smoke && sunOrMoon == 1) then {
+				_marker = "smokeShellPurple" createVehicle getPosATL _crate;
+				_marker setPosATL (getPosATL _crate);
+				_marker attachTo [_crate,[0,0,0]];
+			};
+
+			if (wai_crates_flares && sunOrMoon != 1) then {
```

Comment:
Will not work in arma3 / epoch, its a arma 2 dayz thing, possible replacement "FlareWhite_F"